config.toml

```diff
@@ -5,7 +5,13 @@ port = 3030
 # replace with the account id of the public key you will use to sign relay transactions - this should match the account_id in your json file
 relayer_account_id = "nomnomnom.testnet"   # "relayer.pagodaplatform.near"
 # replace with json files containing 3 entries: account_id (should match relayer_account_id), public_key, private_key
-keys_filenames = ["./account_keys/nomnomnom.testnet.json", "./account_keys/37cb5b936de257d014e663663b186e58e4c3c4c9b4c7643e7b89821e789b0b4b.json", "./account_keys/b278b7752b844a0cc88f8dbdb053b860d6283fb353e7a317b3b82efe01396c3e.json", "./account_keys/d7adc371186bc2013ec5de36be27ed9a36022af1112f7c62530baf6042690f61.json", "./account_keys/df86e9701801e30cf0649b2ea43aeacef60e394e076709d4519880d5a9c7bc9e.json", "./account_keys/e4a6dd159d45d4104679e709d29d49a533e5df9467b311c9dbea7fb8f0077335.json"] # "relayer.pagodaplatform.json"
+keys_filenames = [
+    "./account_keys/nomnomnom.testnet.json",
+    "./account_keys/50407f1203d1cc7eeaa3826838cf0aa77f5c5c656ae0d13d8c8ec6527681aee8.json",
+    "./account_keys/63554bb688a12342ba3bed2a586582ea41fcf028be0d5e10be0b86c8bcef123f.json",
+    "./account_keys/d298bea0d340a103aa9a33920443b6e4ece2788242dfe9425418921a5dba0a9a.json",
+    "./account_keys/d90313b513bd5470ca44205e486f38497c358e800a1a48445e4d2cc096ab163e.json",
+] # "relayer.pagodaplatform.json"
```

Comment:
@anthony-near how come the other keys aren't included? I had to include these new ones otherwise the tests would fail because we wouldn't be able to find the keys

Reply:
I have them included locally. Haven't committed them as you can just change the config to your own keys

Reply:
how about we commit these keys then? Otherwise we'd be loading in the keys at signing time, which isn't ideal since those keys can potentially be wrong (i.e. wrong path), and lead to the service failing down the road instead of immediately when we start it